docs: software catalog items management (#1789)
mia-platform/documentation: docs/software-catalog/manage-items/mia-ctl/update/overview.md

---
id: overview-mia-ctl-update
title:  Update item
sidebar_label:  Update item
---

> ### Update item

Reviewer: We should remove this title since it's the equal to the page's title

```sh
miactl marketplace apply -f myAwesomeService.json
```

You will see the outcome of the operation in the command output:

```sh
1 of 1 items have been successfully applied:

  ID                        ITEM ID             NAME                 STATUS   

  65368hf0c91d871a87afbcbf  my-awesome-service  My Awesome Service   Updated
```

The changes are now reflected to the Console.

### Update of versioned items

Marketplace items of type *Plugin* and *Custom Resources* require a *version* object to determine the differences of the plugin definition over time.
In case you need to update the definition, you can update a versioned item only if you need to change some base fields (more details on the [Create your Company Marketplace](/software-catalog/manage-items/overview.md) page), otherwise you can do so by creating a new version.

You can create a new version of the plugin via [miactl](/cli/miactl/10_overview.md) by simply executing the [`apply`](/cli/miactl/30_commands.md#apply) command, in the same way as explained above, ensuring that the `itemId` and `tenantId` are the same but with a new version, and of course the `resource` object updated.

> As explained in the [Create your Company Marketplace](/software-catalog/manage-items/overview.md) page, we suggest to follow the [Semantic Versioning](https://semver.org/) convention when defining the version of your item.

Reviewer: ```suggestion
As explained in the [Create your Company Marketplace](/software-catalog/manage-items/overview.md) page, you must follow the [Semantic Versioning](https://semver.org/) convention when defining the version of your item.
```
